### DISCUSSION TITLE
Transforming a Template Variable Value with Custom Value

### DISCUSSION THREAD

**Discussion by manojarum** [grafana/grafana#44511] [General]
Hi Team,

 I am using Grafana Version 7.5.9 Integrated with OpenTSDB 2.4.0.  I need help in transforming Template Variable.

Let me give an example. we are working on Front End Development for LTE Performance Management Data in Grafana.

for the Variable MRBTS: the value might be any Integer value i.e 1,2,3,4,5,6,7..  So we need to update a Name instead of integer value. i.e 1 --> Delhi, 2--> Chennai, 3--> Bengaluru ... 

The Integer value will be fetched from OpenTSDB tag_values dynamically and should be replaced with Name which should be fetched using REST API. 

Existing:

![image](https://user-images.githubusercontent.com/68918351/151304533-dbcedeeb-d446-4ff7-bf20-1c0a364eb8e0.png)

Expected:

![image](https://user-images.githubusercontent.com/68918351/151304880-e4c68a8f-322e-43fc-9c5d-81889342de35.png)

Kindly help us in achieving this.

Thanks,
Manoj

**Comment by manojarum** [comment]
Hi,

Any support ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
